lib/engines/content_object_store/app/models/content_object_store/change_dispatcher.rb

```diff
@@ -0,0 +1,15 @@
+module ContentObjectStore
+  module ChangeDispatcher
+    class Now
+      def verb
+        "changed and published"
```

Comment:
This is veeeeery nice!

Reply:
This feels very much like someone's been writing Kotlin recently 😉

Reply:
Thanks! I hope it's not because of Kotlin, I was just trying to practice object design from 99 bottles. 

Before I added the dispatcher I was looking the if conditionals we had in UpdateEdition. We had something like:

```ruby
def call
   …
   if schedule? schedule : publish_now
   …
   flash = schedule? "scheduled" : "changed and published"
   …
```

So I tried to name that behaviour and put it into an injectable thing 🤞